tests/system_test.py

```diff
@@ -748,6 +748,19 @@ def http_addresses(self):
                 ret_val.append("http://%s" % self._cfg_2_host_port(listener))
         return ret_val
 
+    @property
+    def tcp_addresses(self):
+        """Return http://host:port addresses for all tcp listeners"""
+        cfg = self.config.sections('tcpListener')
+        ret_val = []
+        for listener in cfg:
+            require_tls = listener.get("sslProfile")
+            if require_tls is not None:
```

Comment:
`require_tls` name sounds like it is a bool, so maybe it should be a bool

```suggestion
            require_tls = listener.get("sslProfile") is not None
            if require_tls:
```

but that is just nitpick on my side; I guess it looks ok to me and I won't find anything substantial looking at it more